### DISCUSSION TITLE
failed to resolve: use of undeclared type `OrthographicCameraBundle`

### DISCUSSION THREAD

**Discussion by carlosrup** [bevyengine/bevy#6070] [Q&A] [answered]
Hi, I'm following a tutorial, i have this code:

use bevy::prelude::*;

fn setup_camera(mut commands: Commands) {
    commands.spawn_bundle(OrthographicCameraBundle::new_2d());
}
fn main() {
    App::new()
    .add_startup_system(setup_camera)
    .add_plugins(DefaultPlugins)
    .run();
}

I'm getting this error:
failed to resolve: use of undeclared type `OrthographicCameraBundle`

any ideas?

**Comment by hymm** [comment, marked as answer]
It's been renamed to `Camera2dBundle::default()`. see https://bevyengine.org/learn/book/migration-guides/0.7-0.8/#camera-driven-rendering

**Reply by carlosrup** [reply]
Thanks !!
